cmd/hub/mcscontroller/main.go

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 // The controller is responsible for coordinating a common view of exports
 // and imports on a shared (a.k.a. hub) cluster, and replaces the, per-workload-
 // cluster mapping, currently provided by Submariner's lighthouse component.
-package main
+package mcscontroller
```

Comment:
how can you run it as executable if its not in `main` package?

Reply:
need to move the controller and controller test into their own package and leave just the main.go here

Reply:
done in #31